docker-compose-full.yml

```diff
@@ -8,6 +8,7 @@ version: '3.8'
 services:
```

Comment:
docker-compose-full.yaml runs everything in docker.  You can run the groovy integration tests against this, or just point your browser at localhost:3000